src/int/udiv.rs

```diff
@@ -68,3 +72,31 @@ intrinsics! {
         quo_rem.0
     }
 }
+
+#[cfg(any(target_arch = "sparc", target_arch = "sparc64"))]
+intrinsics! {
+    #[win64_128bit_abi_hack]
+    /// Returns `n / d`
+    pub extern "C" fn __udivti3(n: u128, d: u128) -> u128 {
+        u128_divide_sparc(n, d, &mut 0)
```

Comment:
Could this `#[cfg]` happen inside the previous `intrinsics!` block to avoid duplicating the definition?

Reply:
I changed it to use `#[cfg]` inside the previous block, but I don't know if it is beneficial because of all the duplicated `#[cfg]` that is required to completely disable `u128_div_rem`